### ISSUE TITLE
verovio example chopped off

### ISSUE THREAD

**Issue by craigsapp** [music-encoding/guidelines#186]
On the page https://music-encoding.org/guidelines/v4/content/cmn.html

The verovio example in section 4.3.5.5. (Octave Shift) is chopped off at the top of the music:

<img width="924" alt="Screen Shot 2020-08-03 at 3 11 47 PM" src="https://user-images.githubusercontent.com/3487289/89232482-612a5480-d59c-11ea-86a8-f354d576842e.png">

In addition, the XML code underneath the example is not colorized like other examples on the page.

The chopping off of the music is probably related to issue https://github.com/rism-ch/verovio/issues/1553

In other words if that issue is implemented, this example will probably be fixed (unless the top margin option is set manually).

**Comment by rettinghaus** [comment]
The colorisation is different, because this one doesn’t come from an extra file (like all the others), but from the file, that is rendered by Verovio. It uses a different JavaScript/CSS.

**Comment by craigsapp** [comment]
OK: if content is generated dynamically, then the static website generator (Jekyll) cannot process the text to colorize it.

**Comment by musicEnfanthen** [comment]
The example does not seem to show up anymore at all. Can you confirm?

**Comment by lpugin** [comment]
Yes, examples are no longer rendered. We never reached a decision on what to do with them when the guidelines were moved from Jekyll back to ODD.
